### PR TITLE
Login

### PR DESCRIPTION
**Pull request by PragsJ**

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+//import logo from './logo.svg';
+import './App.scss';
+import { Login,Register } from './components/login/index';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLogginActive: true
```
Comment (akshitbansal108) [inline frontend/src/App.jsx]: isLoginActive will be better

File: frontend/src/components/login/style.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          }
+        }
+      }
+    }
+  
+    .footer {
+      margin-top: 3em;
+    }
+  }
+
```
Comment (akshitbansal108) [inline frontend/src/components/login/style.scss]: add an extra blank line at the end of each file

**Comment by akshitbansal108** [comment]
raise pr to develop branch, we will merge the develop branch in master after integration with backend

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,14 +1,73 @@
 import React from 'react';
 //import logo from './logo.svg';
```
Comment (banthianeha) [inline frontend/src/App.jsx]: you can remove this, as well as logo.svg file.

File: frontend/src/components/login/style.scss (hunk 2 above)
Comment (banthianeha) [reply frontend/src/components/login/style.scss]: done

**Comment by akshitbansal108** [comment]
can we delete the serviceWorker.js file

**Pull request merged**